content/overview/quick-start.md

```diff
@@ -17,7 +17,97 @@ You can find the source code for Haystack on GitHub. This is also the main chann
 
 {{< button url="https://github.com/deepset-ai/haystack" text="View Source Code" color="green">}} -->
 
-## Installation
+Haystack is an open source Python framework that helps developers build LLM empowered custom application. In December 2023, a significant update, version 2.0-Beta, was released. This page provides information for both Haystack 1.x and the latest version, 2.0-Beta. For more information on Haystack 2.0-Beta, you can also read the [announcement post](https://haystack.deepset.ai/blog/introducing-haystack-2-beta-and-advent).
+
+## Installation 
```

Comment:
would it make sense to call this 'Installation 2.0-Beta' since we clarify below for the other?

Reply:
@TuanaCelik I wanted make this page future-proof as 2.0-beta will become the default version for Haystack eventually. But I'm fine with changing it to "Installation 2.0-Beta" too